src/main/java/com/minecolonies/coremod/entity/EntityFishHook.java

```diff
@@ -443,12 +444,12 @@ private void updateMotionAndRotation()
      */
     private void checkIfFishBites(final double waterDensity)
     {
-        if (!this.worldObj.isRemote && waterDensity > 0.0)
+        if (!CompatibilityUtils.getWorld(this).isRemote && waterDensity > 0.0)
         {
             int fishingProgressStep = 1;
 
             if (this.rand.nextDouble() < NO_CLEAR_SKY_CHANCE
-                  && !this.worldObj.canBlockSeeSky(new BlockPos(MathHelper.floor_double(this.posX), MathHelper.floor_double(this.posY) + 1, MathHelper.floor_double(this.posZ))))
+                  && !CompatibilityUtils.getWorld(this).canBlockSeeSky(new BlockPos(MathHelper.floor_double(this.posX), MathHelper.floor_double(this.posY) + 1, MathHelper.floor_double(this.posZ))))
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 197 characters long line (which is greater than 180 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00103)